src/librustdoc/html/static/storage.js

```diff
@@ -84,7 +84,7 @@ function onEachLazy(lazyArray, func, reversed) {
 }
 
 // eslint-disable-next-line no-unused-vars
-function hasOwnProperty(obj, property) {
+function hasOwnPropertyRustdoc(obj, property) {
```

Comment:
What was the lint finding that prompted the rename?

Reply:
It was complaining about the fact that `hasOwnProperty` shouldn't be overloaded.

Reply:
This is a polyfill, right? Is it still needed? It looks like it has very wide support: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty.

Reply:
True, since we don't support IE anymore, should be fine to get rid of it now.

Reply:
Like said below, eslint complains about it...